ua/expanded_node_id_test.go

```diff
@@ -106,3 +106,29 @@ func TestParseExpandedNodeID(t *testing.T) {
 		})
 	}
 }
+
+func TestExpandedNodeID_String(t *testing.T) {
+	cases := []struct {
+		name string
+		n    ExpandedNodeID
+		sRep string
+	}{
+		{name: "svr=1;nsu=urn:abc:123;i=3344", n: ExpandedNodeID{ServerIndex: 1, NamespaceURI: "urn:abc:123", NodeID: NewNumericNodeID(1, 3344)}, sRep: "svr=1;nsu=urn:abc:123;i=3344"},
```

Comment:
Maybe put the fields on separate lines since then changes are easier to diff and a bit more in line with the rest of the code.

```go
{
    {
        name: "...",
        n: ExpandedNodeID{...},
        ...
    },
    ...
}
```